movies-manager/src/components/delete-modal/delete-modal.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./delete-modal.component.scss";
+
+const blockName = 'delete-modal';
+
+interface IDeletModalProps {
```

Comment:
Type in interface name. To prevent this in a future use WebStorm or if you are VSCode user use `Code Spell Checker plugin`.

Reply:
done